packages/passport/sdk/package.json

```diff
@@ -61,6 +62,7 @@
   "scripts": {
     "build": "NODE_ENV=production rollup --config rollup.config.js",
     "dev": "rollup --config rollup.config.js -w",
```

Comment:
Can I suggest that we move this script to the Passport sample app please? When the SDK is published it won't contain the sample app, nor will the `concurrently` package be available, so this script may cause confusion for consumers.

Reply:
Yeap sounds like a much better idea. Moved 😄